eclass/java-utils-2.eclass

```diff
@@ -1975,10 +1971,7 @@ etestng() {
 # src_prepare Searches for bundled jars
 # Don't call directly, but via java-pkg-2_src_prepare!
 java-utils-2_src_prepare() {
-	case ${EAPI} in
-		[678]) eapply_user ;;
```

Comment:
Pleases don't remove that condition but drop `6` from it. See https://bugs.gentoo.org/780585#c4

Reply:
@vaukai Please add a comment saying that the plan for > EAPI 8 is to do X then, otherwise it looks like cruft.

Reply:
comment added.